satellitevu/apis/otm.py

```diff
@@ -50,3 +50,95 @@ def get_feasibility(self, *, id: Union[UUID, str]):
             method="GET",
             url=self.url(f"/tasking/feasibilities/{str(id)}"),
         )
+
+    def get_feasibility_response(self, *, id: Union[UUID, str]):
```

Comment:
```suggestion
    def get_feasibility_response(self, *, feasibility_id: Union[UUID, str]):
```

Reply:
In the API docs this is just `id`, so keeping that for now. Might be worth changing the API docs to `feasibility_id` for the feasibility endpoints for consistency with `order_id` for the orders endpoints.